index.html

```diff
@@ -1,15 +1,186 @@
+        <ul type="circle">
+          <li>Мета-теги.</li>
+          <li>Мнемоники.</li>
+          <li>Тэг для работы с изображениями.</li>
+          <li>Теги для работы со списками.</li>
+          <li>Теги для работы с таблицами.</li>
+          <li>Теги для работы с формами.</li>
+          <li>Div-блоки.</li>
+        </ul>
+      </li>
+    </ol>
+    <br/>
+    <br/>
+    <br/>
+    <table>
+      <caption>
+        Таблица 1.
+      </caption>
+      <tr>
+        <th>Tag</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>&lt;table&gt;</td>
+        <td>Defines a table</td>
+      </tr>
+      <tr>
+        <td>&lt;tr&gt;</td>
+        <td>Defines a row in a table</td>
+      </tr>
+      <tr>
+        <td>&lt;th&gt;</td>
+        <td rowspan="2">Defines a &quot;header cell&quot; / &quot;cell&quot; in a table</td>
+      </tr>
+      <tr>
+        <td>&lt;td&gt;</td>
+      </tr>
+      <tr>
+        <td colspan="2">Rarely used</td>
+      </tr>
+        <tr>
+        <td>&lt;caption&gt;</td>
+        <td>Defines a caption table</td>
+      </tr>
+      <tr>
+        <td>&lt;thead&gt;</td>
+        <td>Groups the header content in a table</td>
+      </tr>
+      <tr>
+        <td>&lt;tbody&gt;</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&lt;tfoot&gt;</td>
+      <td>Groups the footer content in a table</td>
+      </tr>
+    </table>
+    <br/>
+    <br/>
+    <br/>
+    <form action:'/' method:'GET'>
+      <fieldset>
+        <legend>My first form</legend>
+        <label for="name">Name</label>
+    <br/>
+        <input id="name" type="text" name="firstName" placeholder="Name" required />
+    <br/>
+    <br/>
+        <label for="email">Email</label>
+    <br/>
+        <input id="email" type="email" name="myEmail" required />
+    <br/>
+    <br/>
+        <label for="pass">Password</label>
+    <br/>
+        <input id="pass" type="password" name="myPassword" required />
+    <br/>
+    <br/>
+    <label for="range">Your programming level</label>
+    <br/>
+        <input id="range" type="range" name="myRange" min="1" max="10" step="1"/>
+    <br/>
+    <br/>
+      <h4>What does HTML stand for?</h4>
+        <input type="checkbox" id="answer1" name="test1" value="1" />
+        <label for="answer1">Hyperlinks and Text Markup Language</label>
+    <br/>
+        <input type="checkbox" id="answer2" name="test2" value="2" />
+        <label for="answer2">Home Tool Markup Language</label>
+    <br/>
+        <input type="checkbox" id="answer3" name="test3" value="3" />
+        <label for="answer3">Hyper Text Markup Language</label>
+    <br/>
+      <h4>What does HTTP stand for?</h4>
+        <input type="radio" name="answer1" value="1"/>Hyper Text Translate Protokol
+    <br/>
+        <input type="radio" name="answer2" value="2"/>Hyper Text Transmission Protokol
+    <br/>
+        <input type="radio" name="answer3" value="3"/>Hyper Text Transfer Protokol
```

Comment:
Text for inputs should be wrapped in `label`.
Inputs `radio` should have the same `name` attr.

Reply:
added label to text inputs, rename name atribute in radiobuttons group